lib/membrane_caps_mp4/mp4.ex

```diff
@@ -0,0 +1,7 @@
+defmodule Membrane.MP4 do
+  @moduledoc """
+  This module provides caps struct for MP4 container buffers.
+  """
+
+  defstruct []
```

Comment:
Assuming we use this as `{RemoteStream, content_format: Membrane.MP4, type: :bytestream}`, is there the need to define struct or only module?

Reply:
Let's have the struct for consistency